tests/test_benchmark.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+from os import path, listdir
+from conftest import skipif
+from devito.exceptions import CompilationError
+from devito import configuration
+from subprocess import CalledProcessError, check_call
+
+
+pytestmark = skipif(['yask', 'ops'])
+
+
+@pytest.mark.parametrize('mode', ['bench'])
+@pytest.mark.parametrize('problem', ['acoustic', 'tti', 'elastic', 'viscoelastic'])
+def test_bench(mode, problem):
+    """
+    Test combinations of modes x problems and created .json filename.
```

Comment:
x -> and

Drop "and created .json filename" (also there was a typo)

Reply:
Sure